README.md

```diff
@@ -1,7 +1,17 @@
 # Popular Movies
 An Android app to allow users to discover the most popular movies playing.
 
```

Comment:
Mention how to add api in the code will be more useful for the other developers to get started easily